BACKLOG.md

```
@@ -85,22 +85,27 @@ Acceptance criteria:
 
 ---
 
-### Define the strict text/font profile
+### Define the strict positioned glyph-run profile
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)